doc/build-windows-msvc.md

```
@@ -0,0 +1,91 @@
+Clone the Dash Core repository to a directory. All build scripts and commands will run from this directory.
+```
+git clone https://github.com/dashpay/dash.git
+cd dash
+```
+
+
+### 4. vcpkg
+
+The presets below reference `$env{VCPKG_ROOT}`. Set it to the vcpkg instance
+shipped with Visual Studio, or to your own clone, before configuring:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)